src/frontend/web_application/src/scenes/UserSecurity/components/LoginDetails/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Button from '../../../../components/Button';
+import TextBlock from '../../../../components/TextBlock';
+import { TextFieldGroup } from '../../../../components/form';
+import './style.scss';
+
+class LoginDetails extends Component {
+  static propTypes = {
+    __: PropTypes.func.isRequired,
+    user: PropTypes.shape({}).isRequired,
+  };
+
+  state = {
+    editMode: false,
+  }
+
+  toggleEditMode = () => {
+    this.setState({ editMode: !this.state.editMode });
```

Comment:
it is not recommended to toggle a state like this, because while the rendering process, the state might not be what we suppose it is.

```js
this.setState(prevState => ({ editMode: !prevState.editMode }));
```